charts/enterprise-logs/templates/_helpers.tpl

```diff
@@ -0,0 +1,61 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
I'd like to revisit all of these functions to understand helm best practices. We inherited these from the Cortex helm chart originally and I'm not sure that the naming convention is valuable, especially as releases make Pod names incredibly long. In our Jsonnet, we use much simpler naming conventions.